pkg/cloud/gcp/assets/validating-admission-policy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: network-tier-annotation-validation-policy
+spec:
+  matchConstraints:
+    resourceRules:
+      - apiGroups: [""]
+        apiVersions: ["v1"]
+        operations: ["CREATE", "UPDATE"]
+        resources: ["services"]
+  validations:
+    - expression: | 
```

Comment:
As above, probably easier if we split this out into several variables and then compose a series of expressions from those, WDYT?

Reply:
yes it looks better.